pkg/storage/model_planStore.go

```diff
@@ -69,6 +73,48 @@ func (s *Store) ModelPlanGetByModelPlanIDLOADER(_ *zap.Logger, paramTableJSON st
 	return planSlice, nil
 }
 
+// ModelPlanTrackingDateGetByModelPlanIDLOADER returns the most recent tracking dates for a series of model plans
+func (s *Store) ModelPlanTrackingDateGetByModelPlanIDLOADER(logger *zap.Logger, paramTableJSON string) (
```

Comment:
maybe this name should reference the task list section name?

Reply:
I'll focus discussion around this on the graphql comment and update this method's name to correspond.